extensions/BMO/template/en/default/pages/etiquette.html.tmpl

```diff
@@ -18,129 +18,107 @@
+  Violations of [% terms.Bugzilla %] etiquette or Mozilla's participation guidelines will be considered 
+  as grounds for curtailing your privileges on this site, or suspending your account altogether.
 </p>
 
+<h3>Commenting</h3>
+
 <ol>
   <li>
-  <span class="heading">No pointless comments</span>.
-  Unless you have something constructive and helpful to say, do not add a
-  comment to a [% terms.bug %]. In [% terms.bugs %] where there is a heated debate going on, you
-  should be even more
-  inclined not to add a comment. Unless you have something new to contribute,
-  then the [% terms.bug %] owner is aware of all the issues, and will make a judgement
-  as to what to do. If you agree the [% terms.bug %] should be fixed, vote for it.
-  Additional "I see this too" or "It works for me" comments are unnecessary
-  unless they are on a different platform or a significantly different build.
-  Constructive and helpful thoughts unrelated to the topic of the [% terms.bug %]
-  should go in the appropriate
-  <a href="http://www.mozilla.org/about/forums/">newsgroup</a>.
+    <span class="heading">No abusing people</span>.
+    Constant and intense critique is one of the reasons we build great products. 
+    It's harder to fall into group-think if there is always a healthy amount of
+    dissent. We want to encourage vibrant debate inside of the Mozilla 
+    community, we want you to disagree with us, and we want you to effectively 
+    argue your case. However, we require that in the process, you criticize 
+    <i>things</i>, not <i>people</i>. Examples of things include: interfaces,
+    algorithms, and schedules. Examples of people include: developers,
+    designers and users. <b>Attacking or encouraging attacks on a person 
+    may result in you being banned from [% terms.Bugzilla %]</b>.  
   </li>
 
   <li>
-  <span class="heading">No obligation</span>.
-  "Open Source" is not the same as "the developers must do my bidding."
-  Everyone here wants to help, but no one else has any <i>obligation</i> to fix
-  the [% terms.bugs %] you want fixed. Therefore, you should not act as if you
-  expect someone to fix a [% terms.bug %] by a particular date or release.
-  Aggressive or repeated demands will not be received well and will almost
-  certainly diminish the impact and interest in your suggestions.
+    <span class="heading">No obligation</span>.
+    "Open Source" is not the same as "the developers must do my bidding."
+    Everyone here wants to help, but no one else has any <i>obligation</i> to fix
+    the [% terms.bugs %] you want fixed. Therefore, you should not act as if you
+    expect someone to fix a [% terms.bug %] by a particular date or release.
+    Aggressive or repeated demands will not be received well and will almost
+    certainly diminish the impact and interest in your suggestions.
+  </li>
+  
+  <li>
+    <span class="heading">No spam</span>.
+    Posting comment spam will lead to the suspension of your account.
```

Comment:
I would beware of making the document too long. Spammers will never say "it's a fair cop, guv", so will either go away because they are bots or because they look for easier targets, or will argue with you - whatever your examples say. All we need is a bit of the guidelines to point to as grounds for suspending their account; we don't need examples.